server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -81,18 +84,29 @@ public class RemoteClusterStateService implements Closeable {
     private static final Logger logger = LogManager.getLogger(RemoteClusterStateService.class);
 
     public static final int INDEX_METADATA_UPLOAD_WAIT_MILLIS = 20000;
+    public static final int GLOBAL_METADATA_UPLOAD_WAIT_MILLIS = 20000;
```

Comment:
total wait time becomes 40secs, we might need to reduce both. maybe give more wait time to global metadata as file is bigger?

Reply:
For now can we keep it as it is, we can revisit the numbers once we work on optimizing the writes of global metadata/index metadata.

Reply:
Yes we can make it as dynamic setting, it will help us tuning this based on latency from remote store as well.

Created issue for making it as dynamic settings: #10688